src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs

```diff
@@ -122,11 +122,18 @@ public IEnumerable<NamedFormulaType> GetFieldTypes()
         {
             return FieldNames.Select(field =>
             {
-                var fieldType = GetFieldType(field);
-                
                 var displayName = DType.TryGetDisplayNameForColumn(_type, field, out var dName)
                     ? dName
                     : null;
+
+                if (this.TryGetBackingDType(field, out var backingDType))
+                {
+                    var t = new TypedName(backingDType, new DName(field));
+                    return new NamedFormulaType(t, displayName);
+                }
```

Comment:
This isn't used, but ideally we should
t
if (this.TryGetBackingDType(field, out var backingDType))
{
t = new TypedName(backingDType, new DName(field));
}
else
{
// ideally code never hits this, but just to be safe
t = new TypedName(ObjNull/Deferred, new DName(field));
}
.
return